root/components/EnterEditNote.js

```diff
@@ -29,9 +29,10 @@ const EnterEditNote = ({
           {exp.l(
             `Entering an {note|edit note} that describes where you got
              your information is highly recommended. Not only does it
-             make it clear where you got your information, but it can also
-             encourage other users to vote on your edit — thus making
-             your edit get applied faster.`,
+             make your sources clear (both now and to users who see
+             the edit years after your edit), but it can also
```

Comment:
this sounds a bit clearer while using less words
```suggestion
             the edit years later), but it can also
```

Reply:
True, thanks!

Reply:
Now it says "years after later" instead of "years later." :)

Reply:
Sigh, I'm an idiot. *Now* it should be ok.